resources/js/beatmap-discussions/new-review.tsx

```diff
@@ -76,31 +76,27 @@ export default class NewReview extends React.Component<Props, State> {
                   {trans('beatmaps.discussions.review.new')}
                   <span className='page-title__button'>
                     <span
```

Comment:
unrelated but this looks like should be a button

Reply:
probably will do that separately (along with the one in `NewDiscussion` 👀